cmake/O2Dependencies.cmake

```diff
@@ -48,15 +48,19 @@ find_package(HEPMC)
 # PR #886 depending on some header files conflict with the IWYU setup
 # disable package for the moment
 #find_package(IWYU)
-find_package(DDS)
 
 find_package(Boost 1.59 COMPONENTS thread system timer program_options random filesystem chrono exception regex serialization log log_setup unit_test_framework date_time signals REQUIRED)
 # for the guideline support library
 include_directories(${MS_GSL_INCLUDE_DIR})
 
 find_package(AliRoot)
 find_package(FairRoot REQUIRED)
-find_package(FairMQ REQUIRED)
+find_package(FairMQInFairRoot)
```

Comment:
Is this a temporary macro for the sake of backward compatibility? I so please add a note about that and when it can be deprecated.

Reply:
It is a renamed version of the already existing FairMQ find module in AliceO2. Yes, it will not be needed any more, once you require a minimum FairRoot version past the split (There is no official release yet, repo split is only in `dev` so far).

Agreed, I will add a comment.

Reply:
Or, if you do not want support for pre-repo-split FairRoot versions, I can immediately remove the `FindFairMQInFairRoot.cmake` module and remove the condition `FairLogger` dependency.

@matthiasrichter @dberzano It is your choice, should I add comments or remove supporting older FairRoot versions?

Reply:
A comment clearly indicating the deprecation. When appropriate we'll get rid of it. It's useful to support both in this hopefully short transition period.

Reply:
Done.